src/requirements_test.txt

```diff
@@ -1,9 +1,12 @@
-fire==0.3.1
 nose2==0.9.2
 pylint==2.5.3
 mock==4.0.2
+fire==0.3.1
 Jinja2==2.11.2
+GitPython==3.1.3
 numpy==1.19.1
 tensorflow==2.1.0
 tqdm==4.48.0
-GitPython==3.1.3
+pydantic==1.6.1
```

Comment:
normally requirements and test requirements dont overlap right? Surely requirements are a subset so this shouldn't be necessary